packages/suite/src/actions/wallet/__tests__/accountActions.test.ts

```diff
@@ -1,10 +1,10 @@
 import configureStore from 'redux-mock-store';
```

Comment:
We should migrate this test to package too. You can mock walletSettingsReducer initial state by hand so it shouldn't be problem.